src/rdkafka_mock.h

```diff
@@ -33,6 +33,9 @@
 #error "rdkafka_mock.h must be included after rdkafka.h"
 #endif
 
+#ifdef __cplusplus
+extern "C" {
+#endif
 
```

Comment:
Can you do the indent trick to keep the IDE from indenting all code to the extern level?
Like this:
https://github.com/edenhill/librdkafka/blob/master/src/rdkafka.h#L53

Reply:
Done. I went ahead and squashed the change into the original commit. I **think** that's what your contributor instructions say.

Reply:
You're acing this PR thing! 👍

Reply:
Thanks the vote of confidence. Luckily I use gitlab on a regular basis, and it's basically a copy/paste of github. Just happy to finally be making my first open source contribution, even a tiny one like this...feels good.